api/agent_test.go

```diff
@@ -1082,6 +1082,20 @@ func TestAPI_AgentChecks(t *testing.T) {
 	}
 }
 
+func TestAgent_AgentChecksRegisterOpts_WithContextTimeout(t *testing.T) {
+	c, err := NewClient(DefaultConfig())
+	require.NoError(t, err)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Nanosecond)
+	t.Cleanup(cancel)
+
+	opts := &QueryOptions{}
+	opts = opts.WithContext(ctx)
+	err = c.Agent().CheckRegisterOpts(&AgentCheckRegistration{}, opts)
+	fmt.Println(err)
```

Comment:
This looks like it is probably a remnant of test debugging.

```suggestion
```

Reply:
Embarassing

Reply:
it happens :shrug: